pkg/sidecar/config/config.go

```diff
@@ -94,8 +100,15 @@ func (s *ScyllaConfig) setupScyllaYAML() error {
 	if err != nil {
 		return errors.Wrap(err, "failed to parse override options for scylla.yaml")
 	}
+	scyllaYAMLConfigMapFilteredBytes := overrideYAMLBytes
+	if len(scyllaYAMLConfigMapBytes) > 0 {
```

Comment:
Perhaps I'm missing something.
Why do we check if the content in non-empty?
Can't we treat it uniformly `mergeYAMLs(scyllaYAMLConfigMapBytes, overrideYAMLBytes)`?

Reply:
You are right. Fixing it.